client/components/plans/plan/style.scss

```diff
@@ -154,3 +154,11 @@
 		@include plans-in-three-columns();
 	}
 }
+
+.jetpack_premium .plan__plan-details ul, .jetpack_business .plan__plan-details ul {
```

Comment:
I think this should probably live inside the `plans/jetpack-plan-details` component CSS.